lightning-background-processor/src/lib.rs

```diff
@@ -224,8 +227,7 @@ impl BackgroundProcessor {
 				// We wait up to 100ms, but track how long it takes to detect being put to sleep,
 				// see `await_start`'s use below.
 				let await_start = Instant::now();
-				let updates_available =
-					channel_manager.await_persistable_update_timeout(Duration::from_millis(100));
+				let updates_available = channel_manager.await_persistable_update_timeout(Duration::from_millis(100));
```

Comment:
Unrelated change can be removed.

Reply:
this comment isn't showing me the affected line numbers for some reason?

Reply:
Line 236 on the latest push.

Reply:
lol now it's showing me the lines, but saying it's 230

Reply:
Yeah, 230 is on the earlier commit, I believe.